cytosnake/helpers/helper_funcs.py

```diff
@@ -0,0 +1,71 @@
+"""
+Module: helper_funcs.py
+
+This module contains helper functions for snakemake workflows to access with. 
+
+"""
+
+
+from pathlib import Path
+from cytosnake.utils.config_utils import load_meta_path_configs
+
+def get_data_folder() -> Path:
+    """_summary_
+
+    Returns
+    -------
+    Path
+        _description_
+
+    Raises
+    ------
+    KeyError
+        Raised if the `data` path is not found in the `_paths.yaml`
+        
+    """
+    meta_configs = load_meta_path_configs()
+    try:
+        data_dir_path = Path(meta_configs["project_dir"]["data"])
+    except KeyError:
+        raise KeyError("Unable to find data folder path")
+
+    return data_dir_path
+
+def get_plate_data() -> str:
+    """Returns absolute path were the data is located
```

Comment:
```suggestion
    """Returns absolute path were the data are located
```

the word "data" is plural :)